docs/modules/ROOT/pages/admin.adoc

```diff
@@ -59,6 +59,18 @@ If, additionally, your contract ABI exposes a `paused()` or `isPaused()` functio
 
 image::defender-admin-pause.png[Defender Admin Paused Contract]
 
+=== Access Control Management
+
+An _access control action_ is a call that's compliant with https://docs.openzeppelin.com/contracts/access-control[OpenZeppelin's Access Control's] `grantRole` and `revokeRole` interfaces via its ABI. 
+
+Defender provides functionality to create such proposals when the ABI contains both `grantRole` and `revokeRole` functions, and you can fill them using the roles that a contract have already indexed before (via events consumed from https://thegraph.com/[TheGraph] if the contract's network is supported, otherwise, indexed via `getLogs` JSON-RPC call). 
+
+In the case you want to provide a role that's not indexed in your contract, you can always provide new roles for your contract by giving it its corresponding `bytes32`.
+
+Also, if the contract is detected to have the Access Control Interface, we provide a small widget to easily navigate contract roles, their admins and their members.
```

Comment:
Let's add a screenshot of the widget, otherwise it is a bit confusing that we mention this here and then show something different right below

Reply:
You're right, I didn't want to add that much since other proposals type descriptions are as extensive as this one.

I added the image!